docs/Help.md

```diff
@@ -0,0 +1,70 @@
+
+### Basic troubleshooting
+Prior to starting on the below, you should first try the basic troubleshooting steps listed here. These steps can resolve a variety of issues:
+
+- Ensure that your Chatterino is up to date.
+- Ensure that you have **BOTH** the x86 and x64 [Visual C++ Redistributables](Resources.md#visual-c-redistributables) installed **AND** have restarted your PC after installation.
+- Try running Chatterino in administrator mode.
+
+### I am having an issue with the Chatterino extension
+Chatterino extension issues should be posted [here](https://github.com/Chatterino/chatterino-browser-ext/issues/new).
+
+### Chatterino not starting/Crashing upon startup
+- Try running Chatterino in administrator mode.
+- Adding Chatterino to your anti-virus/malware whitelist.
+- If nothing else works, try resetting your settings using the `Fresh Install` option in the installer.
+
+### Does using Chatterino accumulate Twitch channel points?
+Unfortunately not. This is a limitation due to Twitch using internal APIs for channel points.<br>
+This will however work with most chatbots, as they simply take you being connected to chat as "watching the stream"
+
+### Can I use Chatterino as my chat overlay for my stream?
+You definitely can, but it's not recommended. We suggest using a service similar to [KapChat](https://nightdev.com/kapchat).
+
+### Why are my messages being sent in webchat, but not in Chatterino?
+There's a few things to cover here:
+
+- If your message is not sent in Chatterino, it is likely not sent in webchat either. Webchat renders messages whether they are sent or not, Chatterino will only render the message if it is accepted by the server.
+- If you're trying to type in a chat where there are a lot of active chatters, then your message may be dropped by Twitch's chat servers (due to the volume )
+- If an alt account has been banned in that chat, you have likely been shadowbanned by Twitch in that channel. Preventing any of your other accounts from speaking in that channel.
+- If you have spammed too much in another chat, you have likely been temporarily shadowbanned by Twitch in channels where you are not: Subscribed, VIP or Moderator.
+
+If all else fails, you may be having [connection issues](#chatterino-is-failing-to-connect-to-chat) to Twitch or your network may be blocking the necessary IRC ports in order to connect.
+
+<!--### Chatterino is failing to connect to chat-->
+
+### Chatterino is repeatedly dis/connecting from Twitch chat
+The most likely issue here is due to the large amount of channels you are connected to. The maximum channel limit is usually around the 100 mark, but may differ from user to user.
```

Comment:
Maybe also mention that user should try closing the number of splits in Chatterino?